torchci/rockset/commons/__sql/test_time_per_oncall.sql

```diff
@@ -44,23 +54,45 @@ With avg_table as (
     date, 
     oncalls
 ), 
+filtered_with_costs as (
+  SELECT 
+    t.avg_duration_in_second as avg_duration_in_second, 
+    t.oncalls as oncalls, 
+    t.date as date, 
+    t.workflow_name as workflow_name, 
+    t.job_name as job_name, 
+    t.test_class as test_class, 
+    t.test_file as test_file, 
+    t.config_job_name as config_job_name, 
+    t.config_shard_name as config_shard_name, 
+    CASE WHEN p.price IS NULL THEN 0 ELSE CAST(p.price AS float) END as price_per_hour 
+  FROM 
+    filtered_table t 
+    LEFT JOIN commons.price_per_config p ON (
```

Comment:
Just curious, which process populates this `commons.price_per_config` collection?

Reply:
I just hard coded it as a csv in rockset. I could make a job to update it nightly though given prices are generally static that may be overkill.